twilio/twilio.ts

```diff
@@ -0,0 +1,11 @@
+const accountSid = process.env.TWILIO_SID
+const authToken = process.env.TWILIO_AUTH_TOKEN
+const client = require('twilio')(accountSid, authToken)
```

Comment:
did it give you an error if you tried to do
```suggestion
import { client } from "twilio" 
client(accountSid, authToken)
```
generally `require` is no longer standard imports and isn't ES6 conformant

Reply:
yeah unfortunately that did not work, so I stuck to the API docs